xla_none.c

```diff
@@ -0,0 +1,20 @@
+/*
```

Comment:
Do we need this file?  Was it included by accident?

Reply:
We do need this file, otherwise the function isn't linked. SIMD uses the same strategy in jsimd_none.c.